src/react-components/info-dialog.js

```diff
@@ -140,14 +156,20 @@ class InfoDialog extends Component {
                   value={this.state.mailingListEmail}
                   onChange={e => this.setState({ mailingListEmail: e.target.value })}
                   className="mailing-list-form__email_field"
-                  required
+                  required={!this.props.allowEmailSkip}
                   placeholder="Your email here"
                 />
-                <label className="mailing-list-form__privacy">
+                <label
+                  className={classNames({
+                    "mailing-list-form__privacy": true,
+                    "mailing-list-form__privacy--hidden":
+                      this.state.mailingListEmail.length === 0 && this.props.allowEmailSkip
+                  })}
+                >
                   <input
                     className="mailing-list-form__privacy_checkbox"
```

Comment:
Maybe use the `%checkbox` styling, for consistency with the rest of the UI? Might have to scale it down to match the text though.